### ISSUE TITLE
how to use show/hide password with password edittext

### ISSUE THREAD

**Issue by adarshakr** [florent37/MaterialTextField#17]
i'am using this library everything is working amazing but i need to use show/hide password with the password edittext can you explain how to use that

**Comment by huzain07** [comment]
yeah, actually we need that feature, cz the newest edittext is support that feature :3

**Comment by stovberpv** [comment]
just using android:inputType attribute
 `
<EditText
                android:id="@+id/password"
                android:layout_width="match_parent"
                android:layout_height="match_parent"
                android:hint="Password"
                android:inputType="textPassword"
                android:textColor="@color/awesomePrimaryDarker" />
`
